docs/device_api.md

```diff
@@ -176,27 +178,31 @@ The example shows there are two differences comparing with provisioning for HTTP
 * The absence of an "endpoint" field
 * The presence of "transport" field, which value has to be "MQTT".
 
-**Registering without endpoint or transport commands**
+**Registering without endpoint commands**
 
-For devices that won't be receiving commands, you don't need to specify neither "endpoint" nor "tranport" fields.
+For devices that won't be receiving commands, you don't need to specify "endpoint" field. In this case, the command will be stored int the IoT Agent waiting for the device to retrieve the commmand. These kind of commnads are called pull commands.
```

Comment:
Is a bit contradictory to say "devices that won't be receiveng commands [meaning use cases in which commands are not involved at all]" and "the command will be stored [thus, command will be used]". Maybe the following wording is better:

> For devices that won't be using commands, you don't need to specify "endpoint" field. This is also the case of devices using commands but in pull mode, in which case the command will be stored int the IoT Agent waiting for the device to retrieve the commmand. These kind of commnads are called pull commands.